### PR TITLE
stability_tester: add bank_mult test and fix exit

### PR DESCRIPTION
**Pull request by Connor1996**

### CODE REVIEW

File: stability-tester/suite/bank_mult.go [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+	count := 0
+	for count < involvedNum {
+		from, to, amount := rand.Intn(c.cfg.NumAccounts), rand.Intn(c.cfg.NumAccounts), rand.Intn(999)
+		if from == to {
+			continue
+		}
+		froms, tos, amounts = append(froms, from), append(tos, to), append(amounts, amount)
+		count++
+	}
+	// make the transfer links to be a loop
```
Comment (siddontang) [inline stability-tester/suite/bank_mult.go]: IMO, the loop is 1 -> 2 -> 3 -> 4 -> 5 -> 1, but here you just link the first and the last accounts.

File: stability-tester/suite/metrics.go [hunk 2]
```diff
@@ -68,6 +68,39 @@ var (
 			Help:      "Counter of failed bank2 verification.",
 		})
 
+	bankMultVerifyDuration = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: "tidb_test",
+			Subsystem: "stability",
+			Name:      "bank_verify_duration_seconds",
```
Comment (disksing) [inline stability-tester/suite/metrics.go]: Choose another name for new test case.

File: stability-tester/suite/bank_mult.go [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+
+func (c *BankMultCase) moveMoney(db *sql.DB) {
+	var (
+		index      string
+		froms, tos []int
+		amounts    []int
+	)
+	count := 0
+	from := rand.Intn(c.cfg.NumAccounts)
+	for count < involvedNum {
```
Comment (disksing) [inline stability-tester/suite/bank_mult.go]: `for count := 0; count < involvedNum; count++ {` or `for len(froms) < involvedNum {`

File: stability-tester/suite/bank_mult.go [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+	for i := 0; i < len(from); i++ {
+		rows, err := selectStmt.Query(from, to)
+		if err != nil {
+			return errors.Trace(err)
+		}
+
+		var (
+			fromBalance int
+			toBalance   int
+			count       int = 0
```
Comment (disksing) [inline stability-tester/suite/bank_mult.go]: just `count int` is enough.

File: stability-tester/suite/bank_mult.go [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+
+	insertStmt, err := tx.Prepare(fmt.Sprintf(`
+		INSERT INTO bank_mult_record (from_id, to_id, from_balance, to_balance, amount, tso)
+			VALUES (?, ?, ?, ?, ?, ?)`))
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	for i := 0; i < len(from); i++ {
+		rows, err := selectStmt.Query(from, to)
```
Comment (disksing) [inline stability-tester/suite/bank_mult.go]: `from[i]` and `to[i]`?

**Comment by disksing** [comment]
LGTM.

**Comment by cwen0** [comment]
This case is to replace the previous `bank case` or both cases all need to run?

**Comment by Connor1996** [comment]
@cwen0 both

**Review by nolouch — APPROVED** [review]
LGTM

**Pull request merged**